.gitmodules

```diff
@@ -18,4 +18,7 @@
 
 [submodule "submodules/ghc-mod"]
 	path = submodules/ghc-mod
-	url = https://github.com/fendor/ghc-mod.git
\ No newline at end of file
+	url = https://github.com/fendor/ghc-mod.git
+[submodule "submodules/apply-refact"]
+	path = submodules/apply-refact
```

Comment:
I think @mpickering has published a new version of this, we can switch to using that now

Reply:
.7 is on GitHub, but not yet on hackage

Reply:
@mpickering can you publish 0.7 to hackage

Reply:
ping @mpickering